src/components/musette/musette.stories.js

```diff
@@ -0,0 +1,136 @@
+import { html } from ".storybook/helpers";
+import { withKnobs } from "@storybook/addon-knobs";
+import lesMusettes from "./musette.js";
+import { useEffect } from "@storybook/client-api";
+
+export default { title: "Carousel/Musette", decorators: [withKnobs] };
```

Comment:
the knobs addon has been buggy. better to just remove it, especially if you don't have any.  we're now slowly migrating to controls/args.